uwblueprint/sistering
# Skip shifts that occur before start date

## Ticket link

Closes #



## Implementation description
* 



## Steps to test
Example using graphql:
```
query Postings {
  postings {
    id
    title
    description
    startDate
    endDate
    shifts {
      startTime
      endTime
    }
  }
}

mutation CreateShifts{
  createShifts(shifts:{postingId:<POSTING_ID>, times:[{startTime:"2022-03-28T01:30", endTime:"2022-03-28T02:00"},{startTime:"2022-03-29T01:30", endTime:"2022-03-29T02:00"} ], startDate:"2022-03-29", endDate:"2022-03-31" recurrenceInterval:NONE}) {
    id
    startTime
    endTime
  }
}
```

Through the UI:
Select "Create Posting" in the navbar on the homepage.
When you get to the page for creating shifts, choose a start date that does not begin on sunday. Add a shift that begins before the start date. 

When submitting the posting, there should be no errors. Check through graphql (using the Postings query above) that shifts before the start date were not generated.



## What should reviewers focus on?
*  Shifts before the start date are skipped while shifts after the start date are not


## Checklist
